### PR TITLE
Removed faculties who left and corrected few google scholar IDs

### PR DESCRIPTION
**Pull request by mohammadarifulalam**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

0. Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-1.csv").

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings-[0-9].csv`._ **Do not modify `csrankings.csv`, which is auto-generated.**

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[0-9].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[0-9].csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

**(Advanced) Quick contribution via a shallow clone** 

A full clone of the CSrankings repository is almost 2GB.  To contribute a change without creating a full local clone of the CSrankings repo, you can do a _shallow clone_.  To do so, follow these steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)
2. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.
3. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to do another contribution and some time has passed, perform steps 1-3 again, creating a fresh fork and shallow clone.

### CODE REVIEW

**Comment by mohammadarifulalam** [comment]
Removed faculties who left and corrected few google scholar IDs

**Comment by mohammadarifulalam** [comment]
@emeryberger I hope you check this issue. It is really important as I found a lot of discrepancies in the list from multiple universities and requested to correct those.

**Review by emeryberger — CHANGES_REQUESTED** [review]
Temporary leave to the federal government = still fully affiliated with the university. That's how that works - she should not be removed.

File: csrankings-1.csv [hunk 1]
```diff
@@ -1066,7 +1066,6 @@ Chris Callison-Burch,University of Pennsylvania,http://www.cis.upenn.edu/~ccb,nv
 Chris Chitty,Massey University,http://www.massey.ac.nz/massey/expertise/profile.cfm?stref=106050,NOSCHOLARPAGE
 Chris Clack,University College London,http://www0.cs.ucl.ac.uk/people/C.Clack.html,6zAjxsIAAAAJ
 Chris Clifton,Purdue University,https://www.cs.purdue.edu/people/clifton,C_cWJIkAAAAJ
-Chris Ding,University of Texas at Arlington,http://ranger.uta.edu/~chqding,q7FfnjgAAAAJ
```
Comment (emeryberger) [inline csrankings-1.csv:1069]: Need to update affiliation rather than removing from the database.

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Review by emeryberger — CHANGES_REQUESTED** [review]
Please restore.

File: csrankings-4.csv [hunk 2]
```diff
@@ -172,7 +172,6 @@ Jia Di,University of Arkansas,https://sites.uark.edu/jdi,NOSCHOLARPAGE
 Jia Hu,University of Exeter,http://emps.exeter.ac.uk/computer-science/staff/jh815,n1Q5NG4AAAAJ
 Jia Jia 0001,Tsinghua University,http://hcsi.cs.tsinghua.edu.cn,NOSCHOLARPAGE
 Jia Li 0003,Beihang University,http://arts.buaa.edu.cn/staff/jiali/index.htm,BfWMlE4AAAAJ
-Jia Liu 0002,Iowa State University,http://web.cs.iastate.edu/~jialiu,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings-4.csv:176]: Change affiliation rather than removing.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-7.csv [hunk 3]
```diff
@@ -284,7 +284,6 @@ Qiang Liu 0001,University of Texas at Austin,https://www.cs.utexas.edu/~lqiang,X
 Qiang Ma 0001,Kyoto University,https://www.db.soc.i.kyoto-u.ac.jp/~qiang/index-e.html,h4AD-WQAAAAJ
 Qiang Ni,Lancaster University,https://www.lancaster.ac.uk/scc/about-us/people/qiang-ni,Pg3lkkgAAAAJ
 Qiang Shen 0001,Aberystwyth University,https://www.aber.ac.uk/en/cs/staff-profiles/listing/profile/qqs,bew3SPIAAAAJ
-Qiang Tang 0005,NJIT,https://web.njit.edu/~qiang,i6oKWhEAAAAJ
```
Comment (emeryberger) [inline csrankings-7.csv:287]: Change affiliation, don't remove.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-5.csv [hunk 4]
```diff
@@ -1496,7 +1496,6 @@ Marta Mejail,University of Buenos Aires,https://www.dc.uba.ar/rrhh/profesores/me
 Marta Ortín-Obón,Universidad de Zaragoza,http://webdiis.unizar.es/~ortin,hyOoV44AAAAJ
 Marta Z. Kwiatkowska,University of Oxford,http://www.cs.ox.ac.uk/marta.kwiatkowska,ArcH6PkAAAAJ
 Marta Zofia Kwiatkowska,University of Oxford,http://www.cs.ox.ac.uk/marta.kwiatkowska,ArcH6PkAAAAJ
-Marten van Dijk,University of Connecticut,https://scl.engr.uconn.edu,byCWPiwAAAAJ
```
Comment (emeryberger) [inline csrankings-5.csv:1502]: Restore. See https://github.com/emeryberger/CSrankings/pull/1264.

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by mohammadarifulalam** [comment]
Will do later

**Pull request closed without merging**